src/config/android/getPackageClassName.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+/**
+ * Gets package's class name (class that implements ReactPackage)
+ * by searching for its declaration in all Java files present in the folder
+ *
+ * @param {String} folder Folder to find java files
+ */
+module.exports = function getPackageClassName(folder) {
+  const files = glob.sync('**/*.java', { cwd: folder });
+
+  const packages = files
+    .map(filePath => fs.readFileSync(path.join(folder, filePath), 'utf8'))
+    .map(file => file.match(/class (.*) implements ReactPackage/))
```

Comment:
It's nice if you re-use it, otherwise doesn't make that much sense to create a variable for one place